mac/app.py

```
@@ -229,29 +289,45 @@ def _update(self, _):
+            alerts.append(("High CPU", f"CPU at {stats['cpu']:.1f}%"))
+        if stats['mem'] >= self.mem_limit and can_notify('mem'):
+            alerts.append(("High Memory", f"Memory at {stats['mem']:.1f}%"))
+        if stats['swap'] >= self.swap_limit and can_notify('swap'):
+            alerts.append(("High Swap", f"Swap at {stats['swap']:.1f}%"))
+        
+        # Add macOS specific ones from core
+        if sys.platform == 'darwin':
+            if stats.get('pressure_status') in ['WARN', 'HIGH'] and can_notify('pressure'):
+                alerts.append(("Memory Pressure", f"Status: {stats['pressure_status']}"))
+            if stats.get('lag_risk') and can_notify('lag_risk'):
+                m = stats.get('macos_mem', {})
```

Ruff (0.14.10)

Local variable `m` is assigned to but never used

Remove assignment to unused variable `m`

(F841)